### PR TITLE
Fix permissions

### PR DESCRIPTION
**Pull request by TriForMine**
**Please describe the changes this PR makes and why it should be merged:**
Fix permissions (sorry my ide changed & to &&)

**Status**

- [ x] Code changes have been tested against the Discord API, or there are no code changes

**Semantic versioning classification:**

- [ ] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: src/util/permissions.ts [hunk 1]
```diff
@@ -74,19 +74,19 @@ export async function botHasPermission(
...
 }
 
 /** Checks if the bot has the permissions in a channel */
-export function botHasChannelPermissions(
+export async function botHasChannelPermissions(
   channelID: string,
   permissions: Permission[],
 ) {
-  return hasChannelPermissions(channelID, botID, permissions);
+  return await hasChannelPermissions(channelID, botID, permissions);
```
Comment (ayntee) [inline src/util/permissions.ts:89]: Since hasChannelPermissions() returns a Promise, you do not have to convert the function to `async`.

Comment (TriForMine) [reply src/util/permissions.ts:89]: It's seems to be needed the function doesn't work without it

Comment (ayntee) [reply src/util/permissions.ts:89]: Confirmed, this does not seem to fix the issue either.

**Review by ayntee — APPROVED** [review]
Thank you, @TriForMine!

**Pull request merged**